test/unit/number/format.js

```diff
@@ -151,6 +151,10 @@ QUnit.test( "should allow rounding", function( assert ) {
 	assert.equal( format( pi, properties( "0.20", en ) ), "3.20" );
 	assert.equal( format( pi, properties( "0.5", en ) ), "3.0" );
 	assert.equal( format( pi, properties( "0.1", en ) ), "3.1" );
+
+	// Handle inaccurate floating point arithmetics like 0.00015 * 10000 = 1.49999999999999.
+	// See #376.
+	assert.equal( format( 0.00015, properties( "0.0001", en ) ), "0.0002" );
```

Comment:
This looks like very little additional coverage relative to the complexity of the changes above. Is there no effect on other methods besides rounding? They all seem to share the same code path.